src/Widgets/ListFooter.vala

```diff
@@ -31,18 +31,26 @@ namespace SwitchboardPlugUserAccounts.Widgets {
 
         construct {
             button_add = new Gtk.Button.from_icon_name ("list-add-symbolic", Gtk.IconSize.SMALL_TOOLBAR);
-            button_add.sensitive = false;
             button_add.tooltip_text = _("Create user account");
 
             button_remove = new Gtk.Button.from_icon_name ("list-remove-symbolic", Gtk.IconSize.SMALL_TOOLBAR);
-            button_remove.sensitive = false;
             button_remove.tooltip_text = _("Remove user account and its data");
 
             get_style_context ().add_class (Gtk.STYLE_CLASS_INLINE_TOOLBAR);
             add (button_add);
             add (button_remove);
 
             button_add.clicked.connect (() => {
+                var permission = get_permission ();
+                if (!permission.allowed) {
+                    try {
+                        permission.acquire ();
+                    } catch (Error e) {
+                        critical (e.message);
```

Comment:
We probably want a visual indication that the operation failed

Reply:
The most common reason this would be triggered is that the user canceled the auto dialog. I don't think we want to show a dialog to say:
```
User dismissed authentication dialog while trying to acquire permission for action-id io.elementary.switchboard.useraccounts.administration
```

Reply:
You can do something like:
```vala
if (!e.matches (GLib.IOError.quark (), GLib.IOError.CANCELLED)) {
    // Show the error
}
```
That way the user don't get a message on cancellation